SchoolProject/src/main/java/Main.java

```diff
@@ -40,12 +48,83 @@ public static void main(String args[]) {
                     var teacher = accessValidator.verifyTeacherAccess(code);
                     if (teacher != null) {
                         access = true;
+                        System.out.println("********************************************");
+                        System.out.println("There are the options available for teachers");
+                        System.out.println("********************************************");
+
+                        System.out.println("Select the option");
+                        System.out.println("a. List the students of ELEMENTARY_SCHOOL grade");
+                        System.out.println("b. Enter a grade for a student");
+                        System.out.println("c. Exit");
+
+                        String optionTeacher = scanner.nextLine();
+
+                        switch(optionTeacher){
+                            case "a":
+                                try{
+                                    students = studentRepository.readStudentsDataFromJsonByGrade(Grade.ELEMENTARY_SCHOOL);
+                                    System.out.println("There are the students of ELEMENTARY_SCHOOL grade");
+                                    int index = 1;
+                                    for (Student student : students) {
+                                        System.out.println(index+":");
+                                        System.out.println("Code : "+student.getCode());
+                                        System.out.println("Name : "+student.getName());
+                                        System.out.println("Age : "+student.getAge());
+                                        System.out.println("Grade : "+student.getGrade());
+                                        System.out.println("Score : "+student.getScore());
+                                        index++;
+                                    }
+                                }catch (Exception ex){
+                                    System.out.println("Error :"+ex.getMessage());
+                                }
+                            break;
+                            case "b":
+                                System.out.println("There are all the students registered in the platform");
+                                students = studentRepository.getAllStudents();
+                                for (Student student : students) {
+                                    System.out.println(student.getCode());
+                                }
+                                System.out.println("Enter the code of the student you wish to modify");
```

Comment:
Maybe all the input messages can go to another class, called ScannerManager or View...